docs/authoring.md

```diff
@@ -11,6 +11,9 @@ The AsyncAPI generator has been built with extensibility in mind. The package us
 1. Templates may contain "filters" or helper functions. They must be stored in the `.filters` directory under the template directory. [Read more about filters](#filters).
 1. Templates may have a configuration file. It must be stored in the template directory and its name must be `.tp-config.json`. [Read more about the configuration file](#configuration-file).
 1. There are params with special meaning. [Read more about special params](#special-params).
+1. Default parameters that you can use in your templates, except the ones that you defined on your own in the config,  are:
```

Comment:
```suggestion
1. The default variables you have access to in any the template file are the following:
```
Think we have to use `variable` instead of parameters, not to confuse it with the configuration parameters.